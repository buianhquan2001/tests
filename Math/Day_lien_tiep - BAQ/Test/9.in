34
32
5 32 26 29 44 6 36 35 50 34 17 9 8 26 21 11 20 1 12 11 18 49 17 4 1 2 39 21 39 9 31 18 
39
47 30 24 44 30 10 28 4 37 31 46 2 38 35 26 14 24 1 16 18 32 32 38 30 32 38 1 14 39 26 25 1 36 45 38 22 38 6 18 
21
38 36 23 13 15 41 44 30 14 29 27 9 12 11 32 39 15 8 47 2 28 
21
21 35 46 19 40 36 23 39 37 45 44 12 11 6 41 49 24 37 29 32 23 
31
37 38 27 6 37 14 1 5 43 30 34 34 3 47 42 20 26 1 10 36 4 41 1 3 27 24 3 12 13 42 21 
37
13 41 3 35 47 38 31 10 29 5 21 23 46 41 35 43 31 5 28 40 43 35 37 17 7 41 30 19 16 44 12 18 47 48 11 42 8 
28
44 23 38 19 4 10 26 46 31 11 18 45 6 20 33 32 31 48 33 41 12 43 7 21 6 31 20 15 
30
12 30 44 2 45 4 7 48 15 49 3 38 47 3 17 30 3 48 1 10 37 10 50 27 48 2 4 35 24 42 
39
35 46 30 3 19 38 37 31 48 32 39 1 15 4 24 46 14 26 7 26 25 11 19 45 47 22 32 20 1 45 16 41 42 24 36 15 12 7 39 
22
17 46 3 22 46 7 26 11 23 43 29 24 5 4 5 15 38 48 11 16 3 41 
38
42 20 1 46 17 25 3 29 14 50 49 18 35 37 37 47 22 17 20 38 33 46 26 25 15 44 10 21 1 47 42 28 39 26 6 42 16 37 
27
36 1 30 36 24 12 32 11 26 25 13 49 5 44 12 30 27 13 7 43 23 48 11 44 16 48 13 
35
12 8 46 43 32 36 18 8 11 38 19 50 22 46 16 23 12 18 25 12 30 28 45 29 43 33 46 23 17 1 42 16 15 2 25 
40
49 3 34 43 9 32 7 42 35 29 33 24 23 41 20 1 42 36 6 22 28 41 16 31 41 34 25 7 26 13 10 31 48 13 21 33 39 36 25 41 
25
49 36 33 3 36 47 31 47 35 30 29 10 39 7 3 42 22 12 50 18 39 49 6 45 47 
28
27 15 20 47 20 23 12 14 7 46 7 20 46 48 18 33 18 36 3 49 31 10 5 3 12 10 29 39 
24
38 12 24 46 15 20 47 11 5 21 9 41 19 6 40 1 14 10 21 35 34 11 50 34 
39
16 7 30 10 32 11 41 25 32 14 48 13 30 8 24 41 23 1 24 48 39 36 5 35 30 7 24 14 24 11 24 1 7 17 12 43 16 41 4 
30
12 33 38 13 39 38 39 48 13 14 15 29 45 44 22 34 36 29 47 49 30 28 23 11 13 12 12 36 2 24 
25
17 11 29 29 5 39 25 45 36 20 11 35 38 1 9 46 37 19 13 12 39 38 9 4 8 
37
14 12 43 29 24 24 17 27 23 8 47 26 31 1 29 48 25 26 47 22 11 36 5 4 28 2 14 40 42 13 27 47 9 49 1 48 44 
29
26 36 2 37 16 39 1 32 13 47 32 47 19 33 50 9 10 22 34 11 34 40 47 9 39 49 46 13 48 
23
11 21 31 19 1 49 12 45 45 1 38 31 1 11 41 4 32 21 20 46 14 42 42 
32
6 23 5 39 1 1 12 48 33 48 37 5 23 36 36 15 17 44 29 4 8 45 32 2 1 8 48 47 21 12 23 5 
33
32 28 45 45 13 19 30 22 5 44 12 27 48 30 46 13 14 29 24 25 28 27 26 24 17 48 37 25 16 45 36 41 20 
38
12 22 43 50 15 19 31 38 15 13 11 17 30 8 9 32 7 24 37 10 3 3 39 29 36 40 40 28 40 18 16 13 38 43 16 6 25 46 
37
19 32 44 16 13 47 13 39 13 25 38 32 6 4 43 6 27 36 21 13 9 48 41 9 17 44 25 30 5 20 34 40 41 31 5 29 33 
34
3 34 29 43 32 16 32 46 50 28 1 48 29 19 2 6 5 21 31 35 24 2 40 24 34 33 30 27 20 13 6 13 4 5 
21
42 20 23 2 34 16 43 19 18 15 23 38 8 2 1 42 21 49 3 31 17 
30
3 30 40 45 14 14 22 2 28 34 35 9 7 44 46 7 33 42 5 48 39 44 18 41 24 49 26 4 6 18 
21
10 45 43 2 28 26 3 36 30 45 42 42 33 43 32 45 29 14 46 45 42 
25
3 27 36 21 37 5 27 24 9 28 30 34 23 3 22 5 44 46 2 30 44 46 16 15 8 
25
41 4 5 30 8 35 31 32 34 43 14 35 34 9 28 45 47 25 32 41 44 20 50 39 2 
26
4 24 21 17 33 27 2 13 18 43 14 9 2 8 35 2 22 16 43 5 10 21 41 35 42 42 
