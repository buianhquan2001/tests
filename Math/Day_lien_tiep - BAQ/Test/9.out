26
43
15
6
47
29
17
53
36
23
40
31
25
32
23
19
27
43
13
20
40
33
38
34
13
18
26
28
27
29
17
26
18
26
